qdrant-landing/content/documentation/guides/configuration.md

```diff
@@ -126,6 +126,9 @@ The following YAML example describes the available configuration options.
 ```yaml
 log_level: INFO
 
+# Configures the format of log lines, supported formats are `text` (default) and `json`
+log_format: text
+
```

Comment:
If we add it here, we should add it to [config/config.yaml](https://github.com/qdrant/qdrant/blob/master/config/config.yaml) first, as this is basically just a copy.

We didn't add it yet as it was still sort of a hidden feature, but I have no objection for adding it now.

Reply:
This also doesn't match the current configuration format, I'll update it accordingly :+1:

Reply:
Added https://github.com/qdrant/qdrant/pull/5943

Reply:
I've opened <https://github.com/qdrant/qdrant/pull/5942> to add the property by default, and have bumped this configuration file.